setup.cfg

```diff
@@ -19,23 +19,12 @@ data-files =
     share/ansible-runner/utils = utils/*
 
 [pep8]
-# E201 - Whitespace after '('
-# E203 - Whitespace before ":"
-# E221 - Multiple spaces after operator
-# E225 - Missing whitespace around operator
-# E231 - Missing whitespace after ','
-# E241 - Multiple spaces after ','
-# E251 - Unexpected spaces around keyword / parameter equals
-# E261 - At least two spaces before inline comment
-# E302 - Expected 2 blank lines found 0
-# E303 - Too many blank lines
-# W291 - Trailing whitespace
-# W391 - Blank line at end of file
-# W293 - Blank line contains whitespace
-ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,W291,W391,W293
+# W503 - Line break occurred before a binary operator
+ignore=W503
 exclude=.tox,venv
 
 [flake8]
+# W503 - Line break occurred before a binary operator
+ignore=W503
 max-line-length=160
-ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,W291,W391,W293,E731,F405
 exclude=.tox,venv
```

Comment:
While I'm on it, let me inform you that https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-extend-exclude exists. Of course, it's best to change it in a separate PR too.

Reply:
I agree it would probably be better to use `extend-exclude`. The default list of `.svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg`